docs/reference/api.rst

```diff
@@ -273,8 +273,8 @@ All functions return a wrapped `ProcessDeviceSupportIn` or
 
         status = mbbIn('STATUS',
             'OK',
-            ('FAILING', alarm.MINOR_ALARM),
-            ('FAILED', alarm.MAJOR_ALARM))
```

Comment:
Was this in the original documentation?  Was a bug if so.

Reply:
Yes